src/Joblify.Core.Api/Login/LoginController.cs

```diff
@@ -1,10 +1,35 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using System.Threading.Tasks;
+using Joblify.Core.Login;
+using Joblify.Core.Login.Dto;
+using Microsoft.AspNetCore.Mvc;
 
 namespace Joblify.Core.Api.Login
 {
-    [Route("api")]
+    [Route("api/[controller]")]
```

Comment:
please use explicit name like:
```suggestion
    [Route("api/login")]
```
in order to not change endpoint after possible refactoring (like class name change)